pkg/providers/aws/cluster_network_provider.go

```diff
@@ -1510,18 +1515,67 @@ func getNetworkProviderConfig(ctx context.Context, configManager ConfigManager,
 		return nil, errorUtil.Wrap(err, "failed to unmarshal aws vpc create config")
 	}
 
-	if vpcCreateConfig.CidrBlock == nil {
-		return nil, errorUtil.New(fmt.Sprintf("CidrBlock in _network strategy tier %s can not be nil", tier))
+	// if the config map is found and the _network block contains an entry, that is returned for use in the network creation
+	if vpcCreateConfig != nil && vpcCreateConfig.CidrBlock != nil && *vpcCreateConfig.CidrBlock != "" {
+		_, vpcCidr, err := net.ParseCIDR(*vpcCreateConfig.CidrBlock)
+		if err != nil {
+			return nil, errorUtil.Wrap(err, "failed to parse cidr block from _network strategy")
+		}
+		cidrMask, _ := vpcCidr.Mask.Size()
+
+		logger.Infof("found vpc cidr block %s/%d in network strategy tier %s", vpcCidr.IP.String(), cidrMask, tier)
+		return vpcCidr, nil
 	}
 
-	_, vpcCidr, err := net.ParseCIDR(*vpcCreateConfig.CidrBlock)
+	//if vpcCreateConfig.CidrBlock is nil or an empty string we can go ahead and set a sensible non overlapping default with a mask size of /26
+	defaultCIDR, err := getNonOverlappingDefaultCIDR(ctx, n.Client, n.Ec2Api, n.Logger)
 	if err != nil {
-		return nil, errorUtil.Wrap(err, "failed to parse cidr block from _network strategy")
+		return nil, errorUtil.Wrap(err, "failed to generate default CIDR")
 	}
-	cidrMask, _ := vpcCidr.Mask.Size()
 
-	logger.Infof("found vpc cidr block %s/%d in network strategy tier %s", vpcCidr.IP.String(), cidrMask, tier)
-	return vpcCidr, nil
+	// a default cidr is generated and updated to the config map, that is returned for use in the network creation
+	return defaultCIDR, nil
+}
+
+// getNonOverlappingDefaultCIDR returns a non overlapping cidr block based on the OpenShift vpc cidr block
+// the default mask is /26
+// for other masks the user is required to provide their own via config
+func getNonOverlappingDefaultCIDR(ctx context.Context, client client.Client, ec2Api ec2iface.EC2API, logger *logrus.Entry) (*net.IPNet, error) {
+	clusterVpc, err := getClusterVpc(ctx, client, ec2Api, logger)
+	if err != nil {
+		errMsg := "failed to get cluster vpc for cidr block"
+		return nil, errorUtil.Wrap(err, errMsg)
+	}
+
+	clusterCIDR := aws.StringValue(clusterVpc.CidrBlock)
```

Comment:
small nit pick, we probably could just pass `aws.StringValue(clusterVpc.CidrBlock)` into `net.ParseCIDR()`